src/partials/auth-security.md

```diff
@@ -24,4 +24,10 @@ If an user is imported into Appwrite with hash differnt than Argon2, the passwor
 Encourage passwords that are hard to guess by disallowing users to pick passwords that contain personal data.
 Personal data includes the user's name, email, and phone number.
 
-Disallowing personal data can be enabled in the Auth service's **Security** tab on the Appwrite Console.
\ No newline at end of file
+Disallowing personal data can be enabled in the Auth service's **Security** tab on the Appwrite Console.
+
+# Session alerts {% #session-alerts %}
+
+Set up email alerts for your users so that whenever another session is created outside of their existing session using their user account information, such as their email address, they will be alerted to the new session.
```

Comment:
> Set up email alerts

There's not much to set up. It's just enabling/disabling so we can just say enable.

> outside of their existing session using their user account information, such as their email address,

This is really long-winded. Best to be short and concise.

```suggestion
Enable email alerts for your users so that whenever another session is created for their account, they will be alerted to the new session.
```